pkg/ottl/ottlfuncs/README.md

```diff
@@ -468,6 +468,7 @@ Available Converters:
 - [FNV](#fnv)
 - [Format](#format)
 - [FormatTime](#formattime)
+- [GetSemconvSpanName](#getsemconvspanname)
```

Comment:
Let's revert this README changes as this function should be documented only on the transform processor's docs.